utils/zerovec/src/ule/plain.rs

```diff
@@ -56,6 +56,23 @@ macro_rules! impl_byte_slice_size {
                 unsafe { core::slice::from_raw_parts_mut(data as *mut Self, len) }
             }
 
+            /// This function can be used for constructing things that can be passed to
+            /// [`ZeroSlice::from_ule_slice()`](crate::ZeroSlice::from_ule_slice) in a const context, avoiding
+            /// parsing checks. `from_ule_slice()` itself cannot be const yet since there are trait bounds,
+            /// however it is not expensive to call in a non const context.
```

Comment:
Nit: Add a docstest showing the intended usage of this function (assigning to a const variable)

Reply:
It's going to be pretty tricky to write a doctest inside a macro